scss/_patterns_notifications.scss

```diff
@@ -205,43 +219,73 @@ $notification-text-margin-bottom: $spv--large - $spv-nudge;
 @mixin vf-notifications-default {
   .p-notification {
     @extend %vf-notification;
-    @include vf-highlight-bar($color-information, left, true);
-    @include vf-icon-info($color-mid-dark);
+    @include vf-highlight-bar($colors--theme--border-information, left, true);
+    @include vf-icon-info(#24598f);
+  }
+
+  // FIXME: temporary workaround until icons have proper theme support
+  // stylelint-disable-next-line selector-max-type -- base styles can use type selectors
+  body.is-dark .p-notification {
+    @include vf-icon-info(hsl(210deg 80% 65%));
   }
 }
 
 // Positive notification styling
 @mixin vf-notifications-positive {
   .p-notification--positive {
     @extend %vf-notification;
-    @include vf-highlight-bar($color-positive, left, true);
-    @include vf-icon-success;
+    @include vf-highlight-bar($colors--theme--border-positive, left, true);
+    @include vf-icon-success(#0e8420);
+  }
+
+  // FIXME: temporary workaround until icons have proper theme support
+  // stylelint-disable-next-line selector-max-type -- base styles can use type selectors
+  body.is-dark .p-notification--positive {
+    @include vf-icon-success(#62a36c, #262626);
   }
 }
 
 // Caution notification styling
 @mixin vf-notifications-caution {
   .p-notification--caution {
     @extend %vf-notification;
-    @include vf-highlight-bar($color-caution, left, true);
-    @include vf-icon-warning;
+    @include vf-highlight-bar(#cc7900, left, true);
```

Comment:
Use colour variables. I guess this is the new value we want to use for caution tinted border, so update it there and use the variable.